.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
+    # Look for `package.json` and `lock` files in the `api` directory
+    directory: "/api"
+    # Check the npm registry for updates every day (weekdays)
+    schedule:
+      interval: "daily"
```

Comment:
I don't believe there is or ever will be a package.json in the `api` service.
Any potential new/ future front-end work where we expect JS or npm to be required would end up as a self-contained `frontend` service

Reply:
I was curious if "`lock` files" meant it would also pickup the [`api/uv.lock`](https://github.com/RMI/pbtar/blob/main/api/uv.lock) file and look for updates of the packages listed there.

Reply:
Seems like there is support for `uv.lock`, but through a different `package-ecosystem` (also it's a very fresh feature as of a few months ago):
https://github.com/dependabot/dependabot-core/issues/10478
https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories#supported-ecosystems-and-repositories

Reply:
🤦🏻 totally mixing up JS and Python